.travis.yml

```diff
@@ -10,4 +10,4 @@ env:
   - ADOPTOPENJDK=8
   - ADOPTOPENJDK=11
 
-script: sbt -Dfile.encoding=UTF8 compilerInterface/mimaReportBinaryIssues headerCheck packageBin doc
+script: sbt -Dfile.encoding=UTF8 compilerInterface/headerCheck jvmfmtCheck dummyBridge/test doc
```

Comment:
`compilerInterface/mimaReportBinaryIssues` isn't necessary?

Reply:
Not for the first version, which I am about to move to a new package.

Reply:
shall I open an issue to remind us to re-enable it?

Reply:
I opened one - https://github.com/scala/compiler-interface/issues/12